README.md

```diff
@@ -4,7 +4,8 @@ The compatibility matrix below assumes the user always install the latest patch
 
 | Plugin | AiiDA | Python | Specification |
 |-|-|-|-|
-| `v1.0 < v2.0` | ![Compatibility for v1.0][AiiDA v2 range] |  [![PyPI pyversions](https://img.shields.io/pypi/pyversions/aiida-optimade)](https://pypi.org/project/aiida-optimade) | ![OPTIMADE API compatibility][OPTIMADE from OPT] |
+| `v1.2 < v2.0` | ![Compatibility for v1.0][AiiDA v2 range] |  [![PyPI pyversions](https://img.shields.io/pypi/pyversions/aiida-optimade)](https://pypi.org/project/aiida-optimade) | ![OPTIMADE API compatibility][OPTIMADE from OPT] |
```

Comment:
I suppose the AiiDA range here should be updated?

Reply:
Ah - maybe it should indeed be changed in the line below as I also suggest [here](https://github.com/aiidateam/aiida-optimade/pull/500#issuecomment-1794529906).